builders/integrations/analyze/dapplooker.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Dapplooker
+description: Build analytics and create dashboards to analyze Moonbeam/Moonriver on-chain data.
+---
+
+# Getting Started with Dapplooker Platform
+
+![Dapplooker Banner](/images/builders/integrations/analyze/dapplooker/subquery-banner.png)
+
+## Introduction {: #introduction }
+
+Developers on Moonriver and Moonbeam can use the [Dapplooker](https://dapplooker.com/) platform to analyze their on-chain data and build dashboards. The integration gives projects the ability to create charts and dashboards to **track their smart contracts** and provide feedback on performance and adoption.
+
+Dapplooker analytics platform complements Moonbeam/Moonriver network by helping users make sense of smart contracts without having to rely on an engineer or analyst. Dapplooker’s intuitive Visual SQL helps browse smart contract data in tabular form and write SQL queries using easy to use editor. Users can create, fork and share dashboards with everyone.
+
+This guide will cover all of the details needed to register your project with the Dapplooker platform and how to build analytics using SQL editor. This guide can be adapted for use on Moonbeam or Moonriver.
+
+
+## Steps to Connect Smart Contracts {: #steps-to-connect-smart-contract}
+
+- Signup or Login to [https://dapplooker.com/login](https://dapplooker.com/login)
+
+  ![Login to Dapplooker](/images/builders/integrations/analyze/dapplooker/dapplooker-1.png)
+
+- After you signup or log in, click on "**My Project**" button in the header section.
+
+- Click "Connect Dapp". Choose "**Connect Smart Contract**" option.
+
+  ![Connect dapp](/images/builders/integrations/analyze/dapplooker/dapplooker-2.png)
+
+- Fill in the details about your Dapp as shown below ✍️.
+- Select network as "**Moonriver" or** "**Moonbeam**".
+- Search and select smart contracts from the autocomplete.
+- Enter your website.
```

Comment:
not sure this step is applicable for smart contracts?

Reply:
Project website helps identify the Dapp and it's domain. It helps us reach out to them for collaboration or any kind of help they need.